src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -244,16 +244,14 @@ private void visitScript(NodeTraversal t, Node script) {
     private void processExports(Node script, String moduleName) {
       if (hasOneTopLevelModuleExportAssign()) {
         // One top-level assign: transform to
-        // /** @const */ var moduleName = rhs
+        // /** @const */ moduleName = rhs
```

Comment:
can you remove const from the comment as well? Sorry, missed that the first time.